390/Lauren_Low.Rmd

```diff
@@ -0,0 +1,40 @@
+---
+title: "census visualization"
+author: "Lauren Low"
+date: "September 16, 2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(tidyverse)
+library(dplyr)
+library(readr)
+library(tsibble)
+library(lubridate)
+library(fpp3)
+```
+
+```{r loading csv and creating census table}
+urlfile1 = "https://raw.githubusercontent.com/390-2020-09-Fall/SCBI-ForestGEO-Data/master/tree_main_census/data/census-csv-files/scbi.stem1.csv"
```

Comment:
Hey Lauren, here is a code review!

Let's leverage the fact that the data already exists in this repo instead of reading it off the web. Also let's be careful with relative file paths: `read_csv(tree_main_census/data/census-csv-files/scbi.stem2.csv)`

https://www.loom.com/share/dabb7a2482084970a98165ee5d3aaefe